tests/unit/test_featuretoggles.py

```diff
@@ -0,0 +1,98 @@
+from typing import List
+
+import pytest
+from pytest_mock import MockerFixture
+
+from ni_measurementlink_service import _featuretoggles
+from ni_measurementlink_service._featuretoggles import (
+    CodeReadiness,
+    FeatureNotSupportedError,
+    FeatureToggle,
+    requires_feature,
+)
+
+RELEASE_FEATURE = FeatureToggle("INCOMPLETE_FEATURE", CodeReadiness.RELEASE)
+NEXT_RELEASE_FEATURE = FeatureToggle("NEXT_RELEASE_FEATURE", CodeReadiness.NEXT_RELEASE)
+INCOMPLETE_FEATURE = FeatureToggle("INCOMPLETE_FEATURE", CodeReadiness.INCOMPLETE)
+PROTOTYPE_FEATURE = FeatureToggle("PROTOTYPE_FEATURE", CodeReadiness.PROTOTYPE)
+
+
+@requires_feature(PROTOTYPE_FEATURE)
+def _prototype_function(x: int, y: str, z: List[int]) -> str:
+    return _prototype_function_impl(x, y, z)
+
+
+def _prototype_function_impl(x: int, y: str, z: List[int]) -> str:
+    return ""
+
+
+def test___current_readiness_level___is_enabled___reflects_readiness_level() -> None:
+    level = _featuretoggles.READINESS_LEVEL
```

Comment:
`from _featuretoggles import READINESS_LEVEL` doesn't work with patching. Perhaps we should expose it as a function instead.